tests/requirements_onnx

```diff
@@ -1,3 +1,3 @@
-numpy>=1.16.6,<1.27
+numpy>=1.16.6
```

Comment:
let us have strict version for dependecies. It will be more clear with what versions we are testing

Reply:
I think I'll align with src/bindings/python/requirements.txt then...